README.md

```diff
@@ -73,6 +73,11 @@ export PATH=$PATH:$GOPATH/bin
 
 (If you run into trouble, see the [Go install instructions](https://golang.org/doc/install)).
 
+#### Prerequisite 2: Install gx and gx-go
+
+You can try using your package manager for it but it is unlikely it is there.
+Other way is to run `make toolkit_upgrade` before running `make install` in the next step.
```

Comment:
maybe go with "currently, the recommended way to install them is to run `make toolkit_upgrade`..."